pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
I've been seeing this get toggled around a lot lately.  6.1 was added on pnpm 8.6.0 and rolled back in 8.6.2, latest version is 8.6.5.   6.0 is the correct lockfile version for us to be using, and we should probably make sure people are not using pnpm 8.6.0 or 8.6.1 to avoid this moving forward